src/2_RTB/AnalisiDeiRequisiti/content.typ

```diff
@@ -0,0 +1,524 @@
+  
+
+/ UC3 : - Visualizzazione del registro dei sensori
+  - *Attore Principale*: utente generico.
+  - *Precondizioni*: almeno un sensore è attivo.
+  - *Postcondizioni*: il sistema mostra un elenco organizzato contenente le informazione relative a tutti i sensori installati. 
+  - *Scenario Principale*:
+    + L'utente accede alla sezione dedicata alla visualizzazione dei sensori tramite la dashboard dell'applicazione;
+    + i sensori sono inizialmente filtrati per tipologia in una visualizzazione tabellare, consentendo una rapida identificazione e analisi.
+  - *Estensioni*: 
+    + selezione un sensore [UC4];
+    + filtraggio dei sensori in base alle loro caratteristiche [UC13].
+
+/ UC4 : - Selezione di un sensore
+  - *Attore Principale*: utente generico.
+  - *Precondizioni*: almeno un sensore è attivo e trasmette dati.
+  - *Postcondizioni*: vengono mostrate le informazioni relative al sensore selezionato e relativi dati storici.
+  - *Scenario Principale*:
+    + L'utente interagisce con la mappa [UC1.1, UC2.1] o, in alternativa, visualizza il registro di sensori [UC3] e seleziona un sensore specifico;
+    + il sistema mostra i dettagli del sensore selezionato.
+   - *Estensioni*: 
+    + modifica dello stato del sensore selezionato[UC5];
+    + impostazione del periodo temporale al fine visualizzare i dati storici relativi a quel periodo e al sensore selezionato sensore selezionato [UC6].
+
+/ UC5 : - Modifica dello #glossary("stato di un sensore")
+  - *Attore Principale*: utente generico.
+  - *Precondizioni*: il sensore è già registrato nella piattaforma.
+  - *Postcondizioni*: le modifiche apportate allo stato del sensore sono state applicate con successo e il sensore riflette le nuove informazioni nello stato operativo.
+  - *Scenario Principale*:
+    + L'utente seleziona il sensore di cui desidera modificare lo stato[UC4];
+    + L'utente modifica lo stato del sensore;
+    + Il sistema registra le modifiche apportate allo stato del sensore.
+  
+
+/ UC6 : - Visualizzazione dati in intervalli seleziati
+  - *Attore principale*: utente generico.
+  - *Precondizioni*: viene selezionato un sensore [UC4].
+  - *Postcondizioni*: il sistema mostra i dati storici relativi all'intervallo temporale selezionato.
+  - *Scenario Principale*:
+    + L'utente accede all'opzione per l'impostazione del periodo temporale per l'analisi storica;
+    + L'utente seleziona l'inizio e la fine del periodo desiderato per l'analisi;
+    + L'utente conferma le impostazioni per il periodo temporale.
+    + Il sistema mostra i dati relativi.
+  - *Scenari secondario*: 
+     + L'utente accede all'opzione per l'impostazione del periodo temporale per l'analisi storica;
+     + L'utente seleziona una data non valida; 
+     + il sistema rifiuta l'input e fornisce un messaggio d'errore.
+  - *Estensioni*:
+       + l'utente analizza i trend nel periodo selezionato[UC10]
+
+
+
+/*
```

Comment:
Possono essere rimossi tutti questi commenti di UC duplicati?

Reply:
Non capisco se devo leggere quelli oppure quelli non commentati

Reply:
no quelli non commentati non serve che li leggi, erano i vecchi casi d'uso, ma visto che ci abbiamo portato dei cambiamenti abbiamo preferito tenerli li commentati in maniera tale che se parlandone si valuta il cambiamento come non adeguato si può recuperarli facile.